### ISSUE TITLE
Do I have to be added as a collaborator before I can push a branch?

### ISSUE THREAD

**Issue by DevonPeroutky** [singer-io/tap-github#80]
I have a fix for #78. However, when I try to push my branch up, I get:
```
devonperoutky@Devons-MacBook-Pro tap-github % git push origin re-add-pr_commits-substream
ERROR: Permission to singer-io/tap-github.git denied to DevonPeroutky.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

I have verified that the remote is correct: 
```
devonperoutky@Devons-MacBook-Pro tap-github % git remote -v
origin	git@github.com:singer-io/tap-github.git (fetch)
origin	git@github.com:singer-io/tap-github.git (push)
```

And my ssh key is working fine:
```
devonperoutky@Devons-MacBook-Pro tap-github % ssh -T git@github.com
Hi DevonPeroutky! You've successfully authenticated, but GitHub does not provide shell access.
```

I even verified that the output of `ssh-add -l -E md5` matches the ssh key attached to my github account.

Unless I'm missing something, I'm almost positive my ssh key is working fine. I'm not sure what else could possibly be the reason that I am unable to push a branch, except a permission issue. This [link](https://help.github.com/en/github/authenticating-to-github/error-permission-to-userrepo-denied-to-other-user) is another thing that makes me think this could be a permission issue, because I have ruled out everything else (I think).

**TL;DR: Is there a way I could be added as a collaborator? I would love to contribute and fix a few bugs because this tap has been really valuable to me.**

**Comment by DevonPeroutky** [comment]
I am an absolute moron. I should have seen everyone was just using forks
